charts/alfresco-search-enterprise/ci/default-values.yaml

```diff
@@ -34,3 +25,9 @@ elasticsearch:
     limits:
       cpu: "1000m"
       memory: "1Gi"
+activemq:
+  fullnameOverride: activemq
```

Comment:
why you dropped resources?

I think at least requests.cpu is always helpful to have to avoid failures related to low cpu availability of the runners

Reply:
I think I tought Iwas fiddling with the unit test values